### PR TITLE
chore: add upgrade-statechannels-latest GH action

### PR DESCRIPTION
**Pull request by andrewgordstewart**
I copied upgrade-statechannels-next and replaced `next` with `latest`.

# [Optional] How Has This Been Tested?
I manually ran [these](https://github.com/statechannels/graph-payments/compare/ags/upgrade-statechannels-latest-action?expand=1#diff-24cac0fc323ba70b408b8ea5bc166f650a6e67b742bbc434134b967b37c07831R34-R38) from the command line, which generated https://github.com/statechannels/graph-payments/commit/c67eac37163622cbf61374a18cbb1e261079b15b

### CODE REVIEW

File: .github/workflows/upgrade-statechannels-latest.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Upgrade statechannels packages to @latest tag
+on:
+  schedule:
+    - cron: '0 8 * * *'
```
Comment (geoknee) [inline .github/workflows/upgrade-statechannels-latest.yml:4]:
```suggestion
on:
```

File: .github/workflows/upgrade-statechannels-latest.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  # enable users to manually trigger with workflow_dispatch
+  workflow_dispatch: {}
```
Comment (geoknee) [inline .github/workflows/upgrade-statechannels-latest.yml:6]: I don't think we want to run this every day?
```suggestion
on:
  # enable users to manually trigger with workflow_dispatch
  workflow_dispatch: {}
```

**Review by geoknee — CHANGES_REQUESTED** [review]
Looks good apart from the scheduling. 

I'm also unsure about the results of your manual test, since there was an unexpected addition to the dependencies on that commit. Can you explain?

File: .github/workflows/upgrade-statechannels-latest.yml (hunk 2 above)
Comment (andrewgordstewart) [reply .github/workflows/upgrade-statechannels-latest.yml:6]: Of course ...

**Review by geoknee — APPROVED** [review]

**Pull request merged**